cli/tests/integration/publish_tests.rs

```diff
@@ -47,6 +47,7 @@ itest!(javascript_missing_decl_file {
   envs: env_vars_for_registry(),
   exit_code: 0,
   temp_cwd: true,
+  http_server: true,
```

Comment:
This was missing and sometimes failing on the CI based on whether a test server was running or not

Reply:
I was so confused by the failing CI 😂